cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,162 @@
+    {cy.get('input#edit-field-override-img-featured-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "thumbnail_image.jpg")
+    {cy.get('input#edit-field-override-img-thumbnail-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "social_media_image.jpg")
+    {cy.get('input#edit-field-override-img-social-media-0-upload').selectFile('@fixture');}
+
+    else if (fileName === "panoramic_image.jpg")
+    {cy.get('input#edit-field-override-img-panoramic-0-upload').selectFile('@fixture');}
+
+    cy.wait(2000);
+    
+})
+
+And('user selects {string} option from {string} dropdown', (dropdown, section) => {
+    cy.get(`.placeholder:contains("${section}")`).parent().find(`input[value="${dropdown}"]`).click({ force: true })
+})
+
+And('user clicks on CROP IMAGE button', () => {
+    cy.get(`span:contains('Crop image')`).click({ force: true })
+})
+
+And('user sets the following crops', (dataTable) => {
+    for (let { crop, cropcase, locator } of dataTable.hashes()) {
+        cy.get(locator).should('contain.text',crop).click();
+        cy.wait(500);
+        cy.get(`div[class*="crop-preview-wrapper"][id='${cropcase}']`).trigger("mouseover").find('span.cropper-face').click({force: true});
+        
+        }
+});
+
+And('user selects {string} Lead Image from the list of images', (image) => {
+    cy.get('span:contains("Lead Image")').parent().click()
+    cy.get('input[name="field_image_article_entity_browser_entity_browser"]').click({ force: true })
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input#edit-name").type(image);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input#edit-submit-cgov-image-media-browser").click();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').click().find("input[name^='entity_browser_select'][class='form-checkbox']").check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true })
+});
+
+
+And('user selects {string} content item', (dropDown) => {
+    cy.get(`input[value='${dropDown}']`).click({ force: true })
+})
+
+And('user clicks on {string} link in the {string} text area', (title, cartOption) => {
+    cy.get(`div.paragraph-type-title:contains('${cartOption}')`).parent().parent().find(`span:contains('${title}')`).parent().click()
+})
+
+And('user clicks on {string} button to select item', (title) => {
+    cy.getIframeBody('iframe[name="entity_browser_iframe_cgov_content_browser"]').find(`input[value="${title}"]`).click({ force: true })
+})
+
+And('user clicks on {string} link in the Internal Link text area within List Items', (linkBtn) => {
+    cy.get(`tbody summary[role='button'] span:contains('${linkBtn}')`).parent().click({ force: true });
+})
+
+
+And('user selects Display from {string} dropdown', (dropDown) => {
+        cy.get('div#edit-field-display-enlarge-wrapper label').should('contain.text',dropDown);
```

Comment:
Can you make "Display" a param instead of hardcoded ?

    And user selects "Display" from "Display Enlarge" dropdown

Reply:
done